opensoc-streaming/OpenSOC-Alerts/src/main/java/com/opensoc/alerts/TelemetryAlertsBolt.java

```diff
@@ -196,11 +195,11 @@ public void execute(Tuple tuple) {
 					LOG.trace("[OpenSOC] Checking alerts cache: " + alert);
 
 					if (cache.getIfPresent(alert) == null) {
-						LOG.trace("[OpenSOC]: Alert not found in cache: " + alert);
+						System.out.println("[OpenSOC]: Alert not found in cache: " + alert);
```

Comment:
yeah it just needs to be removed as it serves no purpose.  the log is already logging the same line. there is no reason to print it out